dll/win32/shell32/dialogs/drive.cpp

```diff
@@ -41,6 +41,7 @@ GetDefaultClusterSize(LPWSTR szFs, PDWORD pClusterSize, PULARGE_INTEGER TotalNum
     DWORD ClusterSize;
 
     if (!wcsicmp(szFs, L"FAT16") ||
+        !wcsicmp(szFs, L"FATX") ||
```

Comment:
Also a reactos hack?

Reply:
No. The boot sector system string is always `FATX` regardless of what kind of format should be performed internally, FATX16 or FATX32 (as you can see [here](https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/lib/fslib/vfatxlib/fatx.c;hb=8248f02ff46abbc5b9a1cec8703c88655cf1813a#l359)).

And now that I'm talking about it I also did a shitty mistake, SHELL32 mustn't care about FATX16/FATX32 differences. I'll refactor this PR as soon as I can.